### PR TITLE
Add Chinese Translation 

### PR DESCRIPTION
**Pull request by yeqing123**
I tested it, no problem！

### CODE REVIEW

File: COLLABORATORS.md [hunk 1]
```diff
@@ -31,3 +30,4 @@
 - [yous](https://github.com/yous)
 - [marocchino](https://github.com/marocchino)
 - [MJayJang](https://github.com/MJayJang)
+- [yeqing](https://github.com/yeqing123)
```
Comment (ccarruitero) [inline COLLABORATORS.md]: Left this file to be modified for maintainers

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 registry.pid
 test
 node_modules
+package-lock.json
```
Comment (ccarruitero) [inline .gitignore]: I think this could in other PR

**Review by ccarruitero — COMMENTED** [review]
Thanks @yeqing123 

The PR looks good, just a minor comments.

I don't speak chinese, so we will need to another chinese speaker can take a look.

File: .gitignore (hunk 2 above)
Comment (yeqing123) [reply .gitignore]: ok

File: COLLABORATORS.md (hunk 1 above)
Comment (yeqing123) [reply COLLABORATORS.md]: ok

**Comment by yeqing123** [comment]
I've made changes.

**Comment by wangding** [comment]
> Thanks @yeqing123
> 
> The PR looks good, just a minor comments.
> 
> I don't speak chinese, so we will need to another chinese speaker can take a look.

Hi ccarruitero, yeqing123's PR looks good. I did some work with him. Please look at here: https://github.com/yeqing123/how-to-npm/pull/1

**Comment by ccarruitero** [comment]
Hi @wangding, thanks for help in the review and sorry for the delay in answer.

I'll squash the commits and merge

**Comment by ccarruitero** [comment]
I added the commit directly in master https://github.com/workshopper/how-to-npm/commit/3b1d8cee8966a106f001570ee23a9d6604364c39

**Pull request closed without merging**